renderdoc/driver/vulkan/wrappers/vk_shader_funcs.cpp

```diff
@@ -29,7 +29,7 @@
 
 RDOC_EXTERN_CONFIG(bool, Replay_Debug_SingleThreadedCompilation);
 
-static RDResult DeferredPipelineCompile(VkDevice device,
+static RDResult DeferredPipelineCompile(VkDevice device, const VkPipelineCache &pipelineCache,
```

Comment:
The vulkan handles don't need to be passed as const ref, they can be simple value passed.